server/queue/queue-rabbitmq/src/test/resources/json/mail_min.json

```diff
@@ -1,16 +1,14 @@
 {
-  "recipients":["recipient1@localhost","recipient2@localhost"],
+  "recipients":[],
   "name":"mail-name-558",
-  "sender":"sender@localhost",
-  "state":"state",
-  "errorMessage":"an error",
-  "lastUpdated":1473344752.000000000,
-  "attributes":{"att1":"rO0ABXQABXZhbHVl"},
-  "remoteAddr":"159.221.12.145",
-  "remoteHost":"toto.com",
-  "perRecipientHeaders":{
-    "recipient1@localhost":{"header":{"X-custom-header":["uedcgukrcg"]}},
-    "recipient2@localhost":{"header":{"X-custom-header-2":["uedcgukrcg"]}}},
+  "sender":null,
+  "state":null,
+  "errorMessage":null,
```

Comment:
oh, now it is :)

Reply:
Sorry I messed up while I already pushed...